x/ibc/03-connection/client/cli/tx.go

```diff
@@ -103,23 +104,26 @@ func NewConnectionOpenTryCmd() *cobra.Command {
 			// TODO: parse strings?
 			counterpartyVersions := args[6]
 
-			proofInit, err := utils.ParseProof(clientCtx.LegacyAmino, args[7])
+			consensusHeight, err := clienttypes.ParseHeight(args[7])
+			if err != nil {
+				return err
+			}
+			proofHeight, err := clienttypes.ParseHeight(args[8])
 			if err != nil {
 				return err
 			}
 
-			proofClient, err := utils.ParseProof(clientCtx.LegacyAmino, args[8])
+			proofInit, err := utils.ParseProof(clientCtx.LegacyAmino, args[9])
 			if err != nil {
 				return err
 			}
 
-			proofConsensus, err := utils.ParseProof(clientCtx.LegacyAmino, args[9])
+			proofClient, err := utils.ParseProof(clientCtx.LegacyAmino, args[10])
 			if err != nil {
 				return err
 			}
 
-			proofHeight := uint64(clientCtx.Height)
```

Comment:
This was incorrect, cannot retrieve `proofHeight/consensusHeight` from our own context

Reply:
thanks for fixing these, most of the cli code is broken and we can address anything else in a final refactor of it